app/views/layouts/content.html.erb

```diff
@@ -91,6 +64,33 @@
             <% end %>
           <% end %>
         <% end %>
+        <% if @front_matter["fullwidth"].blank? %>
+          <aside>
+            <% if @front_matter["jump_links"].present? %>
+              <div class="link-block link-block--jump">
+                <h2 class="link-block__header">On this page:</h2>
+                <ul class="link-block__list">
+                  <% @front_matter["jump_links"].each do |item, anchor| %>
+                    <li><%= link_to(item, anchor) %></li>
+                  <% end %>
+                </ul>
+              </div>
+            <% end %>
+
+            <%= render partial: "layouts/shared/narrow_call_to_action" %>
+
+            <% if @front_matter["related_content"].present? %>
+              <div class="link-block link-block--related">
+                <h2 class="link-block__header">Related content:</h2>
```

Comment:
```suggestion
                <h2 class="link-block__header">Related content</h2>
```
Think we should remove the colon for readability @valentine-carter? Also, our headings don't normally have a colon at the end

(NB this is not to do with the new nav work - this is currently in prod but might as well address it here!)